src/core/crypto_error.c

```diff
@@ -133,99 +134,69 @@ char *crypto_enum_errlist_crypto_cam[] =
         (char*) "CAM_KEYTAB_FILE_KINIT_FAILURE",
 };
 
+/*
+** @brief: Helper Function. Get specific error code, given code, allowable max, and valid string expansion
+** @param: int32_t, int32_t, char*
+ * @return: char*
+*/
+char* Crypto_Get_Crypto_Error_Code_String(int32_t crypto_error_code, int32_t crypto_error_code_max, char* valid_output_string)
```

Comment:
Apologies for confusion -> These two are a little confusing

Reply:
Fixed in upcoming push.  Updated the Max negative error code.